extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -460,28 +460,26 @@ private function isWithinRegistrationPerEmailLimit( $email ) {
 	}
 
 	/**
-	 * @desc Checks if the user/email is not blocked in phalanx and sets the proper response if not
+	 * @desc Checks if the user/email is blocked in phalanx and sets the proper response if so
 	 *
 	 * @param $user
 	 * @return bool
 	 */
-	private function isNotBlockedByPhalanx( $user ) {
+	private function isBlockedByPhalanx( $user ) {
+		$abortError = '';
 
-		return UserLoginHelper::callWithCaptchaDisabled(function($params) {
```

Comment:
`callWithCaptchaDisabled` executes given function with captcha validation disabled; this solution is hacky within of itself, but probably has been used here to get around captcha validation that would otherwise ensue. It should be removed only if this is no longer the case, meaning captcha validation does not happen anymore regardless of this wrapper.